### ISSUE TITLE
Fix/rework Elasticsearchin  Zammad update instructions

### ISSUE THREAD

**Issue by ralf401** [zammad/zammad-documentation#492]

**Comment by AnonymousWP** [comment]
This title doesn't say much on its own. I was encountering the following: documentation tells me to import the 7.x repo and install ElasticSearch, while that's not even working for Zammad 6.x: https://docs.zammad.org/en/latest/prerequisites/software.html#elasticsearch-optional

![image](https://github.com/zammad/zammad-documentation/assets/50231698/95190676-8447-4423-af8a-7cd15be0c47a)

**Comment by ralf401** [comment]
Yes our documentation seems outdated. I created a separate issue for that and will fix it.

**Comment by MrGeneration** [comment]
The table is up to date. Elasticsearch 7.8+ and 8 are supported.
